awx/main/management/commands/create_preload_data.py

```diff
@@ -25,15 +25,17 @@ def handle(self, *args, **kwargs):
                 if not Organization.objects.exists():
                     o, _ = Organization.objects.get_or_create(name='Default')
 
-                    p, _ = Project.objects.get_or_create(
-                        name='Demo Project',
-                        scm_type='git',
-                        scm_url='https://github.com/ansible/ansible-tower-samples',
-                        scm_update_on_launch=True,
-                        scm_update_cache_timeout=0,
-                    )
-                    p.organization = o
-                    p.save(skip_update=True)
+                    # Avoid calling directly the get_or_create() to bypass project update
+                    if not Project.objects.filter(name='Demo Project', scm_type='git').count():
+                        p = Project(
+                            name='Demo Project',
+                            scm_type='git',
+                            scm_url='https://github.com/ansible/ansible-tower-samples',
+                            scm_update_on_launch=True,
+                            scm_update_cache_timeout=0,
+                        )
+                        p.organization = o
+                        p.save(skip_update=True)
```

Comment:
just to verify, we don't need to worry about the organization not being saved here when the `if` is not called, correct?

Reply:
Talking with @shanemcd we decide not to override since the user can do it manually as the project is not a `read-only` entry.

Reply:
Thinking about it, we can fix this issue and also we use the `project` later so  I'll fix it.